static/app/utils/featureFlagOverrides.ts

```diff
@@ -109,10 +115,38 @@ export default class FeatureFlagOverrides {
     return Array.from(names);
   }
 
+  /**
+   * Return list of recently accessed feature flags.
+   */
+  public getFeatureFlags() {
+    return FEATURE_FLAGS;
+  }
+
   /**
    * Stash the original list of features & override organization.features with the effective list of features
    */
   public loadOrg(organization: Organization) {
     organization.features = this.getEnabledFeatureFlagList(organization);
+    // Track names of features that are passed into the .includes() function.
+    const handler = {
+      apply: function (target, orgFeatures, flagName) {
+        // Evaluate the result of .includes()
+        const flagResult = target.apply(orgFeatures, flagName);
+
+        // Check that the flag is not already in the buffer
+        if (!FEATURE_FLAGS.values.some(f => f.flag === flagName)) {
+          // Store the flag and its result in the buffer
+          FEATURE_FLAGS.values[buffer_idx] = {
+            flag: flagName,
+            result: flagResult,
+          };
+          // Increment the buffer index for next time
+          buffer_idx = (buffer_idx + 1) % BUFFER_SIZE;
```

Comment:
with this modulo flags won't be stored in-order. or at least, the start of the 'order' will be unknown.

Consider a case where the buffer size is 10, and 10 items are added. These would all be in-order, with the 1st item in the buffer matching the first flag that was checked.
Now add an 11th item to the buffer, it replaces the item as position 0. So now the earliest flag that was checked is stored in position 1 in the buffer. Listing the items in the buffer from 0 to 9 (10 items total) would list the most-recent flag first, followed by the oldest-flag second.